src/ArrayUtils.f90

```diff
@@ -524,6 +524,8 @@ SUBROUTINE getUnique_1DReal(r,rout,delta,tol)
 
     !Deallocate
     DEALLOCATE(tmpr)
+  ELSE
+    ALLOCATE(rout(0))
```

Comment:
The only thing I can think of is add a comment in the subroutine header to describe when there are 0 unique entries.  Other than comments, looks good.

Reply:
good call.  Just pushed updates